### ISSUE TITLE
about TERN and TERAN results.

### ISSUE THREAD

**Issue by aaaa7788** [mesnico/TERN#11]
In your TERAN paper (table1 and table2), you reported [TERN w. Align] results as higher than [original TERN] and lower than [TERAN], but I noticed that TERN and TERAN have very similar architectures but the shared-transformer configuration for the last TE layers. So if you  train [TERN w. Align] with both TERN and TERAN objectives,  and test it with TERAN inference protocol instead of TERN, shouldn't  it be higher than TERAN?  
Thanks again for you work.

**Comment by mesnico** [comment]
Dear @aaaa7788, although we tried to perform this evaluation, we did not report the results as we measured that the variation in performance is minimal. The idea is that the TERN + TERAN objective acts as a training regularizer. However, we noticed that TERAN is already pretty stable. Therefore, adding the TERN objective as a regularization term, in this case, does not significantly change the final results when testing with the TERAN protocol.
I hope this makes sense to you.
